main.py

```diff
@@ -0,0 +1,25 @@
+from stack import Stack
+from function import Function
+function = Function()
+dataBucketName = "datas3botobucket"
+result_create = function.create_bucket(dataBucketName)
+if result_create:
+    print("Bucket got created successfully")
+else:
+    print("Bucket got created Failed")
+function.create_zipfile('Python')
+result_upload = function.upload_file('lambda_function.zip', dataBucketName)
+if result_upload:
+    print("File  uploaded successfully")
+else:
+    print("File  uploaded Failed")
+
+with open('template.yaml', 'r') as cf_file:
+    cft_template = cf_file.read()
+
+stack_obj = Stack("FirstStacks3lambda", cft_template, "ap-south-1", "rishita10", "datas3bucket")
+stack_obj.create_stack()

```

Comment:
You are everytime calling create_stack function? This will throw an error that the stack is already present. Can you handle this error? And also try to add a waiter so that code will wait till stack is create/update complete

Reply:
Please look into the above comment and change this